### ISSUE TITLE
Eval in https://github.com/udacity/deep-learning-v2-pytorch/blob/master/convolutional-neural-networks/mnist-mlp/mnist_mlp_solution_with_validation.ipynb

### ISSUE THREAD

**Issue by PavelAgurov** [udacity/deep-learning-v2-pytorch#399]
Hi,

In code **https://github.com/udacity/deep-learning-v2-pytorch/blob/master/convolutional-neural-networks/mnist-mlp/mnist_mlp_solution_with_validation.ipynb** when we do validation and test result we should call model.eval(). Right?
Specially because we have DropOut layers.

BR,
Pavel
